### ISSUE TITLE
Backlog

### ISSUE THREAD

**Issue by plexus** [lambdaisland/parseclj#8]
- Syntax features
  - [ ] namespaced maps
  - [ ] metadata
  - [ ] #=(...)
  - [ ] #?{}
  - [ ] quote / backquote / unquote / splice
  - [ ] var literal #'
- parseedn
  - [ ] lossless output format
  - [ ] unify string/buffer usage (optional first string arg)
  - [ ] raise errors on non-EDN syntax
- parseclj-ast
  - [ ] implement parseclj-ast-value
- General
  - [ ] Add more docstrings
  - [ ] propagate lex errors when :fail-fast

**Comment by volrath** [comment]
@plexus what about adding zippers for AST traversal? or do you feel it'd be out of this project scope?

**Comment by plexus** [comment]
I'd be interested to see AST zippers. I do feel most of the above is more essential though.

**Comment by volrath** [comment]
@plexus Right. I wasn't trying to make any statement on priorities, just wondering about the fact that this should be included into the backlog.

By the way, what do you mean about "implement parseclj-ast-value"? Is that a selector that will return the value of a node as an elisp data structure, or as a AST (node) data structure?

I think I could start by making the project a bit more MELPA-compliant in a few days. I'll let you know.
